src/components/List/List.vue

```diff
@@ -35,6 +35,11 @@ export default {
         return ListProperties.length === 0;
       },
     },
+    /** Set header cell, th tag, min width MD / 150px  */
+    thWidth: {
+      default: true,
+      type: Boolean,
+    },
```

Comment:
Se for true, todos os lugares que consomem este componente herdarão como default esta prop, é isto mesmo?

Acho que poderia dar um nome para a prop mais intuitivo, minWidthTitle.
Na documentação eu colocaria em qual campo será aplicado, no caso o Title

Reply:
Isso, esse min width já esta padrão. Mas para adequar a um layout preciso do componente sem o min width.